docs/configure_admin_workstation_post_install.rst

```diff
@@ -16,19 +16,19 @@ protect access to the most sensitive assets in the SecureDrop system:
 #. SSH on the *Monitor Server*
 
 The installation process blocks direct access to each of these assets, and sets
-up `Authenticated Tor Onion Services`_ (ATHS) to provide authenticated access
-instead. Authenticated Tor Onion Services share the benefits of Tor Hidden
-Services, but are only accessible to users who possess a shared secret
-(``auth-cookie`` in the Tor documentation) that is generated during the hidden
+up `authenticated onion services`_ to provide authenticated access
+instead. Authenticated onion services share the benefits of regular onion services,
+but are only accessible to users who possess a shared secret
+(``auth-cookie`` in the Tor documentation) that is generated during the onion
 service setup process.
 
-In order to access an ATHS, you need to add one or more "auth-cookie" values
-to your Tor configuration file (``torrc``) and restart Tor. Doing this manually
-is annoying and error-prone, so SecureDrop includes a set of scripts in
-``./tails_files`` that can set up a Tails instance to automatically
-configure Tor to access a set of ATHS. In order to persist these changes across
-reboots, the Tails instance must have persistence enabled (specifically, the
-"dotfiles persistence").
+In order to access an authenticated onion service, you need to add one or more
+"auth-cookie" values to your Tor configuration file (``torrc``) and restart Tor.
+Doing this manually is annoying and error-prone, so SecureDrop includes a set of
+scripts in ``./tails_files`` that can set up a Tails instance to automatically
+configure Tor to access a set of authenticated onion services. In order to
+persist these changes across reboots, the Tails instance must have persistence
+enabled (specifically, the "dotfiles persistence").
```

Comment:
`dotfiles` is used for the desktop shortcuts but not for the config for persistent access to authenticated services, the bit in parentheses could be dropped.

Reply:
:heavy_check_mark: Done in 54af89906dd20db0b41053b9c8dc765750c77827

Reply:
:heavy_check_mark: